tests/test_fusor.py

```diff
@@ -254,14 +254,14 @@ def transcript_segment_element():
             "label": "TPM3",
             "type": "GeneDescriptor",
         },
-        "transcript": "refseq:NM_152263.3",
+        "transcript": "refseq:NM_152263.4",
         "element_genomic_end": {
             "id": "fusor.location_descriptor:NC_000001.11",
             "label": "NC_000001.11",
             "location": {
                 "interval": {
-                    "end": {"type": "Number", "value": 154170400},
-                    "start": {"type": "Number", "value": 154170399},
+                    "end": {"type": "Number", "value": 154170401},
+                    "start": {"type": "Number", "value": 154170400},
```

Comment:
@korikuzma @katiestahl I'm unsure if this coordinate should be increased by 1? According to UTA the start of exon 8 for NM_152263.4 is 154170399, but I think there may have been discussion earlier about whether this value was 0- or 1-indexed

Reply:
Ok, in that case I think the coordinates shouldn't be increased by as the value for `alt_start_i` in UTA for exon 8 is 154170399?

Reply:
I'll look into this more, but I changed these test cases because they were failing and off by one.
![Screenshot 2024-07-11 at 1 53 41 PM](https://github.com/user-attachments/assets/ba25e6ff-8ff8-42eb-a1d7-487839eb7c54)

Maybe it's because I removed this code since this issue was fixed in Cool-Seq-Tool? I thought we didn't need this code in fusor anymore, but maybe we still do